### ISSUE TITLE
I sugget my Idea 

### ISSUE THREAD

**Issue by yimchang** [leozide/leocad#136]
When copying, it is copied to the coordinates of the block to be copied, and it is hard to confirm the copy confirmation visually.

May I proceed to make it visible by changing the coordinates one space?

**Comment by leozide** [comment]
I agree that there's no clear indication that the copy operation worked, but the problem with moving pieces is that you don't know where people want to place the copy. For example, if you always move the copy above the original then when people want to place it on the side of the original piece they will have to move it down first and then move it to the side, adding an extra step.

Can you think of any other ways of showing the copy?

**Comment by yimchang** [comment]
I think the issue of direction is not important. 
After changing the coordinates of an existing block, 
How about temporarily changing the color of the block?

**Comment by j6t** [comment]
Is this going to be addressed in some way? Or can this be closed? If the issue remains, it needs a better title.
